docs/language/aggregates/max.md

```diff
@@ -39,7 +42,31 @@ yield max(this)
 4
 ```
 
-Unrecognized types are ignored:
+Maximum of several string values:
+```mdtest-spq
+# spq
+max(this)
+# input
+"bar"
+"foo"
+"baz"
+# expected output
+"foo"
+```
+
+Determining a maximum among mixed numeric and string inputs is not yet possible:
```

Comment:
I would change this to say something like feeding `max` a mix of string and numeric values results in an error.